quick-guide-for-developers.md

```diff
@@ -0,0 +1,54 @@
+
+# The Way We Work: Quick Guide for Developers
+
+## Your Role
+
+Your role is to use software engineering best practices to support rapid and safe delivery of high-quality software.
+
+## Your Focus
+
+At this stage in your career you should focus on your own personal success and professional development, while doing what you can to help others to succeed.
+
+## Your Responsibilities
+
+Your key responsibilities are to:
+
+- Carry out **Feature Development** work within a Delivery Team 
+- Complete **20% Time** work within each Delivery Team sprint
+- **Pair** with other members of a Delivery Team where this would be productive
+- **Buddy** with newer developers to help them settle into their role
+
+Details on how these responsibilities should be carried out can be found in The Way We Work.
+
+## How You Will Be Supported
+
+Your line manager will:
+
+- meet with you on the first day of each new sprint
+- ensure you have appropriately challenging work to undertake
+- help you record your achievements and plan for upcoming challenges
+- help you choose your own Objectives & Key Results (OKRs)
+- ensure you are using 20% Time to make continuous progress towards your OKRs
+
+Within a delivery team:
+
+- there will be an understandable system design
+- there will be documentation to help you in unfamiliar territory
+- Senior Developer(s) will mentor (train) you in new areas
+- Senior Developer(s) will ensure you follow Engineering Quality Framework standards and best practices
+- you will have the opportunity to pair with others
+- your code will be reviewed promptly by other developers
+- significant issues will be escalated to and handled by SDMT
+
+## How You Will Support Others
+
+To support a delivery team you will:
```

Comment:
To support a delivery team you should:

or

To support a delivery team you are expected to:

or

To support a delivery team you are required to: